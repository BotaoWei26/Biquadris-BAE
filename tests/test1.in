drop
2clock
drop
drop
3ri
drop
3right
drop
6rgith
6right
drop
counter
counter
counter
3ri
drop
right
right
right
drop
100right
drop
clock
drop
clock
5right
drop
clock
100right
drop
6right
clock
drop
drop
drop
drop
drop
drop
drop
drop
drop
drop
drop
drop
drop
100right
drop
drop
100right
drop
clock
6right
left
drop
100right
drop
clock
4ri
drop
100right
drop
right6
6rigth
6ri
2left
clock
drop
100right
drop
clock
100right
drop
clock
4right
drop
clock
100right
drop
clock
6right
drop
clock
100right
drop
3right
drop
100right
2left
drop
