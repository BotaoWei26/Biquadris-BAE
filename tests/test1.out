

 Level:    1      Level:    1
 Score:    0      Score:    0
 -----------      -----------
|...........|    |...........|
|...........|    |...........|
|..L........|    |...........|
|LLL........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
 -----------      -----------
Calling drop


 Level:    1      Level:    1
 Score:    0      Score:    0
 -----------      -----------
|...........|    |...........|
|...........|    |...........|
|...........|    |TTT........|
|...........|    |.T.........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|..L........|    |...........|
|LLL........|    |...........|
 -----------      -----------
Calling clockwise


 Level:    1      Level:    1
 Score:    0      Score:    0
 -----------      -----------
|...........|    |...........|
|...........|    |...........|
|...........|    |.T.........|
|...........|    |TTT........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|..L........|    |...........|
|LLL........|    |...........|
 -----------      -----------
Calling drop


 Level:    1      Level:    1
 Score:    0      Score:    0
 -----------      -----------
|...........|    |...........|
|...........|    |...........|
|OO.........|    |...........|
|OO.........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|..L........|    |.T.........|
|LLL........|    |TTT........|
 -----------      -----------
Calling drop


 Level:    1      Level:    1
 Score:    0      Score:    0
 -----------      -----------
|...........|    |...........|
|...........|    |...........|
|...........|    |..L........|
|...........|    |LLL........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|OO.........|    |...........|
|OOL........|    |.T.........|
|LLL........|    |TTT........|
 -----------      -----------
Calling right


 Level:    1      Level:    1
 Score:    0      Score:    0
 -----------      -----------
|...........|    |...........|
|...........|    |...........|
|...........|    |.....L.....|
|...........|    |...LLL.....|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|OO.........|    |...........|
|OOL........|    |.T.........|
|LLL........|    |TTT........|
 -----------      -----------
Calling drop


 Level:    1      Level:    1
 Score:    0      Score:    0
 -----------      -----------
|...........|    |...........|
|...........|    |...........|
|J..........|    |...........|
|JJJ........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|...........|    |...........|
|OO.........|    |...........|
|OOL........|    |.T...L.....|
|LLL........|    |TTTLLL.....|
 -----------      -----------